_docs/quickstarts/scanner/default-rules.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Default Rules
+order: 106
+---
+
+# {{ page.title }}
+
+Forseti Scanner is preconfigured with some default rules. When any of these rules' conditions are met, 
```

Comment:
nit: A couple of rules are here to really serve as examples on how to define the rules (rather than best practices).  So, to be clear to the user, can we add write this sentence as:
```Forseti Scanner is preconfigured with some default rules to serve as examples how rules can be written.```


* Service accounts should not be granted the role `Folder Admin`.
* all gmail users are allowed to be members of my G Suite groups